src/navUI.js

```diff
@@ -1,5 +1,6 @@
-// eslint-disable-next-line import/extensions
-import navItems from './navItems.js';
+import navItems from './navItems';
+
+import restaurantLogoImage from '../assets/images/restaurant-logo.png';
```

Comment:
Webpack will import the file, and when building production mode, replace `restaurantLogoImage` with the path to the image.